### DISCUSSION TITLE
社交媒体与抑郁 | /cn/2018/02/social-media-depression/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1570] [Comments]
有一位圣地亚哥州立大学心理学教授基于数据分析称《青少年心理健康恶化的原因可能是社交媒体》。在这篇文章发表的同一天，牛津大学的一位心理学家驳斥了她的结论，并在几个月前写了一篇文章称《社交媒体不像吸毒》。
[&hellip;] 我认同这位牛津心理学家对圣地亚哥心理学教授的数据分析的反驳，我认为这是很标准的科学研究范式，值得学习；即使主观看法再强烈，也不要强行用数据来确认自己的成见。社交媒体是否导致青少 &hellip;

https://yihui.org/cn/2018/02/social-media-depression/

**Comment by MqCreaple** [comment]
这个想法很有意思。作为一位高中生，我的社会经验可能还不太能支撑我理解您这个观点背后的理由。以我个人经验来看，我身边的很多人都是您所说的“社交媒体成瘾者“，我自己也有时会因社交媒体耽误自己的学习。

我主要是想质疑两点：

1. 您提出的这些标准（”睡眠时间和质量，作业时长和质量，看书时长，注意力集中的能力，对外部社会有积极意义的行为次数“）这些真的能代表更健康的生活吗？恐怕大部分现代人（包括青少年和成年人）都有至少一两条标准不符合吧
2. 我们是否应当尊重那些所谓的”社交媒体成瘾者“他们自己的判断和意愿？如果他们认为当下的享乐是正确的、值得推崇的，我们是否要因为这件事”对社会没有价值“而去否定他们的判断。虽然我自己不想成为这样的人，但考虑到”社交媒体成瘾“的社会危害性不大，我觉得没有必要做过多干涉吧？

**Reply by yihui** [reply]
如果高中生都能像你这样思考，那我可能也就没什么好担心的了。

1. 健康与不健康之间肯定不存在一个明确的分界线，所以有一两条不符合也没什么；况且我觉得也不存在一种普适的健康标准，文中我只是试探性举了几个可能的准则。成瘾这件事是不是健康生活的障碍，取决于它占了生活多大比例。比如，如果一个人本来需要八小时的睡眠，但其中三小时是在床上醒着刷社交媒体（生理上困了但心理上兴奋），我觉得这个就不怎么健康了。
2. 问题就在于危害性多大在当前还不好说。人是社会动物，互相之间会有牵连，一个人的行为会对其他人产生影响。如果我们都像老虎一样是独居动物，那我觉得就随个人便了，爱怎么玩怎么玩，我绝对尊重任何人的判断和意愿。

其实我更在意和反对的是社交媒体的制造者，而不是使用者。
